test/performance/performance-tests.sh

```diff
@@ -42,6 +42,7 @@ function update_knative() {
   # and services from istio before reintalling it, to get them freshly recreated
   kubectl delete deployments --all -n istio-system
   kubectl delete services --all -n istio-system
+  kubectl create namespace "${SYSTEM_NAMESPACE}"
```

Comment:
Do we create the same NS elsewhere? It seems like if not, it couldn't pass the test.

Reply:
Hmm could you explain what do you mean by `elsewhere`?
I just realized the `SYSTEM_NAMESPACE` could have been created when this command is run, so I changed it to `kubectl get ns "${SYSTEM_NAMESPACE}" || kubectl create namespace "${SYSTEM_NAMESPACE}"` to only create the namespace if it does not exist.

Reply:
My understanding is that SYSTEM_NAMESPACE is where we install knative components. If we don't create somewhere else, this change is the first line to create the NS. It seems not right as the test passed before (I think?). So there should be somewhere else creating the NS. Yeah my concern is that whether we care if the NS has been created.

Reply:
The performance testing setup has been failing for a while, and I tried several times to fix it but now it still has issues (either because failed to completely fix it or because others broke it). The last fix was in https://github.com/knative/serving/pull/9447, and it's now failing with `namespaces "knative-serving" not found` - https://prow.knative.dev/view/gs/knative-prow/logs/ci-knative-serving-recreate-clusters/1306610254414352384#1:build-log.txt%3A134, so this is the correct fix for it.